packages/design-system/src/Collapsible/Collapsible.types.ts

```diff
@@ -0,0 +1,28 @@
+import { TextProps } from "Text";
+
+export type CollapsibleProps = {
+  /** The title of the collapsible component. */
+  title: string;
+  /** The content to be displayed when the component is expanded. */
+  children: React.ReactNode;
+  /**
+   * Whether the component is initially expanded or collapsed.
+   * Defaults to true.
+   */
+  isOpen?: boolean;
+  /** Callback function to be called when the component is expanded or collapsed. */
+  onHeaderClick?: (isOpen: boolean) => void;
+  /** Additional CSS classes for the collapsible container. */
+  className?: string;
+  /** Whether to show the icon or not. */
+  showIcon?: boolean;
```

Comment:
Do we need this flag? if it's being passed then we will show else not. Won't that approach work?

Reply:
We are showing arrow-down and arrow-up by default. If there's a use case where we don't want to show the default icons and custom icons, this will come in handy.